src/features/orders/ordersSlice.ts

```diff
@@ -374,9 +374,7 @@ export const take = createAsyncThunk(
         dispatch(submitTransaction(transaction));
       }
     } catch (e: any) {
-      console.error(e);
-      // TODO: don't throw the actual error message, just show a helpful message
-      dispatch(declineTransaction({ reason: "", hash: "" }));
+      dispatch(declineTransaction(e.message));
```

Comment:
reverted due to consistency with the rest of the file. Also looked at where this is actually handled and it only logs to the console (hence removing the additional log). AFAIK this is hit if the user cancels the metamask popup, so doesn't need to trigger any UI action.